internal/domain/aboutus/handler.go

```
@@ -38,11 +40,26 @@ func (h *handler) submitHandler(c *gin.Context) {
 	img, _ := c.FormFile("image")
```

do not ignore the error. It would be better to handle it.